openqabot/loader/repohash.py

```diff
@@ -21,11 +21,15 @@ def get_max_revision(
 ) -> int:
     max_rev = 0
 
-    url_base = f"http://download.suse.de/ibs/{project.replace(':',':/')}"
+    url_start = "http://download.suse.de/ibs"
```

Comment:
Maybe we can somehow derive this from `OBS_URL` somehow or introduce another variable?